docs/book/basics/status_subresource.md

```diff
@@ -0,0 +1,127 @@
+## Status Subresource 
```

Comment:
This should be a title, so we should use one `#`. i.e. `# Status Subresource`

Reply:
Done